sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -283,8 +273,8 @@ azure:
         scopes: {scope1}, {scope2}
 ``` 
 
-### Protect the Resource API through Resource Server
-Please refer to [azure-spring-boot-sample-active-directory-spring-oauth2-resource-server][resource-server] for access resource api.
+### Protect the resource APIs in webapi
```

Comment:
We should move this section along with obo one ahead of the `Authenticate in frontend` section since we're deprecating the filters.

Reply:
We should find a better word here, instead of webapi.

Reply:
Or we could use `resource server`?